.github/workflows/test.yml

```diff
@@ -66,5 +66,27 @@ jobs:
         go-version: '1.21'
     - name: Checkout code
       uses: actions/checkout@v3
-    - name: Test
-      run: make test-community
+    - name: Set up gotestsum
+      run: |
+        go install gotest.tools/gotestsum@latest
+        mkdir -p tmp/test-results
+    - name: Test 
+      run: |
+        CGO_ENABLED=0 go test -timeout=5m $(shell go list ./... | grep -v /vendor/ | grep -v pkg/detectors | grep -v 'pkg/sources')
```

Comment:
You need to run tests with gotestsum and if you provide the go test command to it then it must have json output